src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerExecutor.cs

```diff
@@ -454,7 +454,7 @@ private void ExecuteCompilationActionsCore(ImmutableArray<CompilationAnalyzerAct
                 analyzerStateOpt?.ProcessedActions.Add(operationAction);
```

Comment:
Actually, I noticed that you already did so for 2., so probably doing it for 1. shouldn't be too much of an effort. Thanks John!